### PR TITLE
Add a managed workspace for RH services

### PR DESCRIPTION
**Pull request by gbenhaim**

### CODE REVIEW

**Comment by gbenhaim** [comment]
depends on https://github.com/redhat-appstudio/tenants-config/pull/61

File: cluster/stone-prd-rh01/managed/rh-managed-mng-s-2-tenant/release_strategy.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: ReleaseStrategy
+metadata:
+  name: push-to-external-registry-strategy
+spec:
+  params:
+    - name: extraConfigGitUrl
+      value: https://github.com/hacbs-release/strategy-configs.git
```
Comment (scoheb) [inline cluster/stone-prd-rh01/managed/rh-managed-mng-s-2-tenant/release_strategy.yaml:8]: do you have a PR for this?

File: cluster/stone-prd-rh01/managed/rh-managed-mng-s-2-tenant/release-pipeline-sa.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: release-service-account
```
Comment (scoheb) [inline cluster/stone-prd-rh01/managed/rh-managed-mng-s-2-tenant/release-pipeline-sa.yaml:4]: we should include the RemoteSecrets as shown here:

https://github.com/redhat-appstudio/tenants-config/blob/main/cluster/stone-prd-rh01/managed/rh-managed-rhtap-ser-tenant/remote-secrets/remote_secret-quay-token.yaml

**Review by scoheb — CHANGES_REQUESTED** [review]
some questions

File: cluster/stone-prd-rh01/managed/rh-managed-mng-s-2-tenant/release_strategy.yaml (hunk 1 above)
Comment (gbenhaim) [reply cluster/stone-prd-rh01/managed/rh-managed-mng-s-2-tenant/release_strategy.yaml:8]: not yet

Comment (gbenhaim) [reply cluster/stone-prd-rh01/managed/rh-managed-mng-s-2-tenant/release_strategy.yaml:8]: https://github.com/hacbs-release/strategy-configs/pull/17

File: cluster/stone-prd-rh01/managed/rh-managed-mng-s-2-tenant/release-pipeline-sa.yaml (hunk 2 above)
Comment (gbenhaim) [reply cluster/stone-prd-rh01/managed/rh-managed-mng-s-2-tenant/release-pipeline-sa.yaml:4]: done

**Comment by gbenhaim** [comment]
depends on https://github.com/hacbs-release/strategy-configs/pull/17

**Review by scoheb — APPROVED** [review]

**Pull request merged**